lib/python/tests/test_auth.py

```diff
@@ -0,0 +1,82 @@
+import os
+
+import pytest
+from dotenv import load_dotenv
+
+from ..bailoclient.auth import CognitoSRPAuthenticator
+from ..bailoclient.config import APIConfig, BailoConfig, CognitoConfig
+from ..bailoclient.utils.exceptions import UnauthorizedException
+
+load_dotenv()
+
+
+@pytest.fixture()
+def authenticator():
+    return cognito_authenticator()
+
+
+def cognito_authenticator():
+    cognito_config = CognitoConfig(
+        user_pool_id=os.environ["BAILO_USERPOOL"],
+        client_id=os.environ["BAILO_CLIENT_ID"],
+        client_secret=os.environ["BAILO_CLIENT_SECRET"],
+        region=os.environ["BAILO_REGION"],
+    )
+
+    config = BailoConfig(
+        cognito=cognito_config, api=APIConfig(url="http://bailo.com", ca_verify=True)
```

Comment:
Can you find & replace all instances of `bailo.com`, which might be an address that exists, with either `localhost:8080` or `example.com`.